sgt-back-end/server.js

```diff
@@ -0,0 +1,143 @@
+import pg from 'pg';
+import express from 'express';
+
+const app = express();
+app.use(express.json());
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const getAllGrades = `select *
```

Comment:
You only use this once and you don't use this approach with the other SQL. Maybe this should just go down below in the function that uses it.